pacai/student/qlearningAgents.py

```diff
@@ -23,109 +23,60 @@ class QLearningAgent(ReinforcementAgent):
 
     Functions you should use
         - self.getLegalActions(state) which returns legal actions for a state
-    """
 
-    def __init__(self, index, **args):
-        """
-        You can initialize Q-values here...
-        """
 
-        super().__init__(index, **args)
+    Methods to Implement:
 
-    def getQValue(self, state, action):
-        """
-        Returns Q(state,action)
+    - def getQValue(self, state, action):
+        - Returns Q(state,action)
         Should return 0.0 if we never seen
```

Comment:
Make line consistent in length (i.e. there does not need to be a break between this line and then next)